.github/workflows/publish-command.yml

```diff
@@ -301,14 +293,16 @@ jobs:
           DOCKER_HUB_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
           # Oracle expects this variable to be set. Although usually present, this is not set by default on Github virtual runners.
           TZ: UTC
-      - name: Finalize Sentry release
-        if: startsWith(matrix.connector, 'connectors')
+      - name: Create Sentry Release
+        if: startsWith(matrix.connector, 'connectors') && success()
         run: |
-          sentry-cli releases finalize "${{ env.IMAGE_NAME }}@${{ env.IMAGE_VERSION }}"
+          SENTRY_RELEASE_NAME="airbyte-${{ env.IMAGE_NAME }}@${{ env.IMAGE_VERSION }}"
```

Comment:
for now I've just hardcoded `airbyte-` but this will likely need a rethink at the point where we're not just having `airbyte-` connectors. Since we're not there yet it feels superfluous to add logic for that already.